de_workloads/ingest/Ingest_AzureSql_Example/data_factory/adf_triggers.tf

```diff
@@ -0,0 +1,23 @@
+resource "azurerm_data_factory_trigger_tumbling_window" "trigger_Ingest_AzureSql_Example" {
+  name            = "trg_AzureSql_ExampleSource"
```

Comment:
This is probably not too important, but in other resources we tend to keep the same terraform resource id (trigger_Ingest_AzureSql_Example) as the resource name (trg_AzureSql_ExampleSource), so maybe consider having the same name for both?